.github/workflows/ci.yaml

```diff
@@ -11,57 +11,42 @@ on:
 jobs:
   ci:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        otp-version: [24.3]
-        elixir-version: [1.13]
 
     services:
       db:
-        image: postgres:12
+        image: postgres:14
         env:
           POSTGRES_DB: elixir_boilerplate_test
           POSTGRES_PASSWORD: boilerplate
         ports: ["5432:5432"]
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
     env:
+      ERLANG_VERSION: 25.0
+      ELIXIR_VERSION: 1.13
       MIX_ENV: test
       DATABASE_URL: postgres://postgres:boilerplate@localhost/elixir_boilerplate_test
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - uses: erlef/setup-elixir@v1
         with:
-          elixir-version: ${{ matrix.elixir-version }}
-          otp-version: ${{ matrix.otp-version }}
+          otp-version: ${{ env.ERLANG_VERSION }}
+          elixir-version: ${{ env.ELIXIR_VERSION }}
 
-      - uses: actions/setup-node@v1
+      - uses: actions/cache@v3
         with:
-          node-version: 16.13
+          path: |
+            deps
+            _build
+            priv/plts
+          key: ${{ runner.os }}-mix-${{ env.ERLANG_VERSION }}-${{ env.ELIXIR_VERSION }}-${{ hashFiles(format('{0}/mix.lock', github.workspace)) }}
```

Comment:
@remi @JoeyBG what do you guys think about this? I can not say it's simpler/cleaner this way 🤔